src/components/network/filter-creation-panel.tsx

```diff
@@ -0,0 +1,252 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
```

Comment:
this component is not used any more, should be deleted.
src/components/network/map-selection-creation.tsx should have been created with git mv filter-creation-panel.tsx map-selection-creation.tsx

but it's not a big deal